### ISSUE TITLE
lumen 5.7 not work

### ISSUE THREAD

**Issue by mowangjuanzi** [mnabialek/laravel-sql-logger#18]
I followed the tutorial to register for service providers

```php
$app->register(Mnabialek\LaravelSqlLogger\Providers\ServiceProvider::class);
```

I found that the component didn't work properly

The reason is that lumen does not automatically register the event object

So if you change the registration code to the following:

```php
$app["db"]->connection()->setEventDispatcher($app["events"]);
$app->register(Mnabialek\LaravelSqlLogger\Providers\ServiceProvider::class);
```

Lumen v5.8 is untested.

So I created issus just to provide a solution. After looking for a long time before looking for a solution, I did not find the one I wanted after looking for the source code.

**Comment by mnabialek** [comment]
Thank you for report. I will try to investigate this issue.

**Comment by mowangjuanzi** [comment]
got it

**Comment by mnabialek** [comment]
@mowangjuanzi I've tested it in Lumen 5.8 and in fact queries were not logged. I've made a change and just released 2.2.2 version and it should work now. Please verify and confirm it's fine now.

**Comment by mowangjuanzi** [comment]
@mnabialek  Perfect. It works.
